pants.toml

```diff
@@ -28,5 +25,5 @@ interpreter_constraints = [">=3.9"]
 enabled = true
 repo_id = "BB6CBFC17A66FAB02E3FA88C13113D25"
 
-[sendwave-docker]
-report_progress = true
\ No newline at end of file
+[docker]
+build_verbose = true
```

Comment:
This option prints build output to the console (you have to scroll up to see it though):

https://github.com/pantsbuild/pants/blob/852f420212ecd29ff54cd4a906bf80944ec272bd/src/python/pants/backend/docker/subsystems/docker_options.py#L158-L162